packages/web-ui/src/ds/atoms/Button/index.tsx

```diff
@@ -85,6 +91,11 @@ const buttonVariants = cva(
         variant: 'nope',
         className: 'p-0',
       },
+      {
+        variant: 'shiny',
+        size: 'small',
+        className: 'rounded-lg text-xs px-2',
```

Comment:
Isn't text size already defined in size? Why does this have a smaller size?

Reply:
Could be a way of doing any already existing variant shiny? this looks super harcoded